src/Razor/Microsoft.AspNetCore.Razor.Language/src/TagHelperDescriptorCache.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public static class TagHelperDescriptorCache
```

Comment:
```suggestion
    internal static class TagHelperDescriptorCache
```

Reply:
Actually, why have this in the language  repo at  all? Seems like  more of a tooling concern given I don't see t his  used  in this PR. Ditto witht he memory cache impl

Reply:
Won't that make it package internal to `Microsoft.AspNetCore.Razor.Language` and inaccessible in aspnetcore-tooling based namespaces?